content/95-ismms-himc-appendix.md

```diff
@@ -0,0 +1,6119 @@
+also varied considerable across the countries tested and may reflect
+another important confounder. The authors could consider analyzing the
+Case Fatality Rate (CFR, % of patients with COVID-19 that die), to more
+correct for exposure although testing availability will still bias this
+result. Variability in mortality within countries or cities with
+variable vaccination and similar exposure could also be appropriate
+although confounders will still be present.
+
+#### Significance
+
+BCG vaccine is a live attenuated strain derived from *Mycobacterium
+bovis* and used for a vaccine for tuberculosis (TB). This vaccine has
+been proven to be efficient in preventing childhood meningitis TB, but
+doesn’t prevent adult TB as efficiently. For this reason, several
+countries are now only recommending this vaccine for at-risk population
+only.
+
+**This study shows that there is a correlation between BCG vaccination
+policy and reduced mortality for Covid-19**. Indeed, BCG vaccine has
+been shown to protect against several viruses and enhance innate
+immunity [@doi:10.1016/j.cmi.2019.04.020], which could explain why it could protect against SARS-CoV-2
+infection, but the exact mechanism is still unknown. **Moreover, the
+efficiency of adult/older people vaccination and protection against
+Covid-19 still needs to be assessed.** Regarding this, Australian
+researchers are starting a clinical trial of BCG vaccine for healthcare
+workers [@url:https://www.mcri.edu.au/BRACE], to assess if it can protect them against Covid-19.
+
+#### Credit
+
+*This review was undertaken as part of a project by students, postdocs and faculty at the Immunology Institute of the Icahn School of Medicine, Mount Sinai.*
+
+
+### Non-neural expression of SARS-CoV-2 entry genes in the olfactory epithelium
+
+[@doi:10.1101/2020.03.25.009084]
+
+#### Keywords
+
+- CoVID-19
+- 2019-nCoV
+- SARS-CoV-2
+- anosmia
+- olfaction
+- scRNAseq
+
+
+#### Main Findings
+
+-   Study analyzed bulk and scRNAseq data of olfactory cell types from
+    publicly-available mouse, nonhuman primate and human datasets.
+
+-   show that *ACE2* and *TMPRSS2* (genes involved in SARS-CoV-2 entry)
+    are expressed in olfactory epithelial (OE) cells, basal stem cells
+    and respiratory epithelium (RE), but not sensory neurons.
+
+-   Comparison of human RE and OE datasets (Deprez et al. 2019; Durante
+    et al. 2020) revealed that *ACE2* and *TMPRSS2* expression in OE
+    sustentacular cells was similar to expression in the remainder of
+    the non-nasal respiratory tract.
+
+#### Limitations
+
+-   Transcript data alone from healthy respiratory/olfactory cells is
+    not sufficient to confirm infectivity of nasal passage, or to
+    indicate damage to epithelia.
+
+-   No mechanism defined for anosmia; it is not clear if epithelial
+    injury leads to reduced sensitivity or increased inflammation and
+    altered immune contexture drives neural/epithelial dysfunction. Will
+    be critical t
```

Comment:
```suggestion
#### Limitations
```

Reply:
This is not showing up correctly, very weird. Basically there is a "Limitations" with a `###` header (rather than `####`)